packages/api/src/utility.ts

```diff
@@ -166,6 +166,7 @@ export const MAX_COMMENT_LENGTH = 1000
 export const ONE_HOUR_IN_MILLISECONDS = 60 * 60 * 1000
 export const ONE_DAY_IN_MILLISECONDS = 24 * ONE_HOUR_IN_MILLISECONDS
 export const FIFTEEN_MINUTES_IN_MILLISECONDS = 900000
+export const ONE_MONTH_IN_MILLISECONDS = 31 * ONE_DAY_IN_MILLISECONDS
```

Comment:
Won't monthly billing be inconsistent with a static 31 days, or is this how it's meant to be?

Reply:
@penk85 I just need it to compare if roughly a month has passed. I went for 31 because that means it's a month for sure.